content/en/integrations/guide/service-account-impersonation.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Set up service account impersonation and automatic project discovery for Google Cloud
+kind: guide
+further_reading:
+    - link: 'integrations/google_cloud_platform/'
+      tag: 'Documentation'
+      text: 'Google Cloud Platform Integration'
+    - link: 'https://cloud.google.com/iam/docs/granting-changing-revoking-access'
+      tag: 'Google Cloud'
+      text: 'Manage access to projects, folders, and organizations'
+---
+
+Instead of using static credentials in private key files, you can use [service account impersonation][1] and automatic project discovery to integrate Datadog with [Google Cloud][2].
+
+This solution enables you to monitor all projects visible to a service account by assigning the required roles in the relevant projects. You can assign these roles to projects individually, or you can configure Datadog to monitor groups of projects by assigning these roles at the organization or folder level. Assigning roles in this way allows Datadog to automatically discover and monitor all projects in the given scope, including any new projects that may be added to the group in the future. 
+
+<div class="alert alert-warning">
```

Comment:
if we are releasing this to GA next week, we can remove this and update the documentation on June 1st.